src/sim65/6502.c

```diff
@@ -234,6 +257,8 @@ int PrintCycles;
             }                                                   \
             TEST_CF (Regs.AC);                                  \
             SET_OF ((res < -128) || (res > 127));               \
+            if (CPU!=CPU_6502)                                  \
```

Comment:
According to the style in this file this would be `CPU != CPU_6502` (with spaces).